wiki/installing-and-updating-drivers.md

```diff
@@ -85,12 +85,12 @@ See [Reinstalling GPU drivers from the OEM's website](reinstalling-gpu-drivers#m
 
 ## Troubleshooting 🔧
 ### If you can access Windows:
-a. You can verify if all of your drivers are signed.
+#### You can verify if all of your drivers are signed.
 1. Press <kbd>Win</kbd> + <kbd>R</kbd> and write `sigverif` and press <kbd>enter</kbd>.
 
 2. It will pop a window to verify and you press the <kbd>Start</kbd> button.
```

Comment:
Maybe change it to "A window will pop up. Press the <kbd>Start</kbd> button to verify."?

Reply:
```suggestion
2. A window will pop up. Press the `Start` button to verify driver signatures.
```